stdlib/contrib/bonitoo-io/tickscript/tickscript.flux

```diff
@@ -0,0 +1,95 @@
+package tickscript
+
+import "experimental"
+import "influxdata/influxdb"
+import "influxdata/influxdb/monitor"
+import "influxdata/influxdb/schema"
+import "universe"
+
+// alert
+alert = (
+    check,
+    id=(r)=>"${r._check_id}",
+    message=(r)=>"Check: ${r._check_name} is: ${r._level}",
+    details=(r)=>"",
+    crit=(r) => false,
+    warn=(r) => false,
+    info=(r) => false,
+    ok=(r) => true,
+    tables=<-) =>
+  tables
+    |> drop(fn: (column) => column =~ /_start.*/ or column =~ /_stop.*/)
```

Comment:
Can this be `drop(columns: ["_start", "_stop"])` instead?

Reply:
This also drops _start and _stop time columns on joined input.